### ISSUE TITLE
Element not highlighted on focus

### ISSUE THREAD

**Issue by smcmurtry** [veteransaffairscanada/vac-benefits-directory#1758]
From Julianna:

2.4.7 Focus Visible
With keyboard navigation it is important that the focused component can be visually identified. If the focused component is not somehow highlighted, the user cannot be sure which component has focus.

Make sure that user interface elements, that can receive keyboard focus, are highlighted on focus.

Occurrences on this page1

<body>
Skip to Main ContentFrançaisFrBETATh

*Notes: How do I meet this criteria?*

This is tripping a failure for some reason with how its coded. It appears to usually focus. We should discuss potential flags for this as we will be looking into automated testing in the next shorty while.

Use one of the following techniques from WCAG 2.0:

F78 Failure of Success Criterion 2.4.7 due to styling element outlines and borders in a way that removes or renders non-visible the visual focus indicator
G15 Using a tool to ensure that content does not violate the general flash threshold or red flash threshold
G149 Using user interface components that are highlighted by the user agent when they receive focus

**Comment by SupeDeDupe** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Accessibility issues](https://trello.com/c/knKtuM4T/100-accessibility-issues)

**Comment by smcmurtry** [comment]
As far as I can tell, the skip to main content button is always visible when it's focussed (it has a `:focus` style that makes it visible). I'll close this for now.
